core/services/workflows/syncerlimiter/limiter.go

```diff
@@ -93,12 +117,26 @@ func (l *Limits) Decrement(owner string) {
 // getPerOwnerLimit returns the default limit per owner if there are no overrides found
 // for the given owner.
 func (l *Limits) getPerOwnerLimit(owner string) int32 {
+	addr := common.HexToAddress(owner).String()
 	if l.perOwnerOverrides == nil {
 		return l.config.PerOwner
 	}
-	limit, found := l.perOwnerOverrides[owner]
+	limit, found := l.perOwnerOverrides[addr]
 	if found {
+		l.lggr.Debugw("overriding limit for owner", "owner", addr, "limit", limit)
 		return limit
 	}
+
+	l.lggr.Debugw("did not find owner in overrides, returning default", "owner", addr, "limit", l.config.PerOwner)
 	return l.config.PerOwner
 }
+
+// normalizeOverrides ensures all incoming keys are common address strings
+func normalizeOverrides(in map[string]int32) map[string]int32 {
+	out := make(map[string]int32)
+	for k, v := range in {
+		addr := common.HexToAddress(k).String()
```

Comment:
@MStreet3 We can't have eth specific types seeping in here -- can we just normalize by lowercasing the origin + comparison value instead?